includes/admin/page-optimizer/src/app/page-optimizer/components/icons/icon-svg.tsx

```diff
@@ -249,14 +249,56 @@ export const SettingsLine: FC<SvgProps & { width?: number, category: string }> =
     );
 };
 
-export const TitanLogo:FC<SvgProps> = ({cls}) => (
-    <svg className={`${cls}`} width={width} height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
-        <path d="M20.2937 0H9.70633C4.34567 0 0 4.34567 0 9.70633V20.2937C0 25.6543 4.34567 30 9.70633 30H20.2937C25.6543 30 30 25.6543 30 20.2937V9.70633C30 4.34567 25.6543 0 20.2937 0Z" fill="#2E223D"/>
-        <path d="M13.5798 8.20254L10.5266 16.6329L13.5798 16.6937L11.0431 23.0886L19.0937 13.6709L15.9038 13.5949L19.4734 8.14178L13.5798 8.20254Z" fill="white"/>
+
+export const TestModeLine: FC<SvgProps & { width?: number, category?: string }> = ({ width = 400, category }) => {
```

Comment:
@zakirox123 This file is already bloated. Let's create individual files for each icon.